CMakeLists.txt

```diff
@@ -80,6 +80,13 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc64|powerpc64)")
     if(${OQS_DIST_BUILD})
         set(OQS_DIST_PPC64_BUILD ON)
     endif()
+elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "(ppc|powerpc)")
+    message(WARNING "There is currently no CI for: " ${CMAKE_SYSTEM_PROCESSOR})
```

Comment:
I do not quite understand this warning: Isn't there ppc CI via Travis, @bhess ? But then again, that CI link in the README is dead. Created #1827 to track.

Reply:
For the warning, I just reproduced what other cases had. If unneeded, perhaps should be removed accordingly.

Reply:
No, I'm OK with what you did: Just checking again, I indeed don't find PPC CI and our PLATFORMS.md file does not document such. So, please disregard my comment.